packages/babel-plugin/src/utils/build-compiled-component.ts

```diff
@@ -1,6 +1,7 @@
 import generate from '@babel/generator';
 import template from '@babel/template';
 import * as t from '@babel/types';
+import { ax } from '@compiled/react/runtime';
```

Comment:
Not sure if a babel plugin should import stuff from runtime. I've been thinking about moving `ax` to `@compiled/utils`. But I noticed `@compiled/utils` requires some modules (e.g. source-map) which shouldn't exist in runtime. Although bundler might remove it from runtime code, I think it's still dangerous for runtime to depend on `@compiled/utils`.

It's safe other way around. (build time library depends on runtime library)

Reply:
I think this is ok as we don't import from the babel plugin in the react package. 

As for side-effects in the runtime accidentally being imported, we should be safe if we use the `size-limit` tool to prevent regressions.